synapse/models/inet.py

```diff
@@ -1801,22 +1807,14 @@ def getModelDefs(self):
                     )),
 
                     ('inet:email:message:attachment', {}, (
-                        ('message', ('inet:email:message', {}), {
-                            'ro': True,
-                            'doc': 'The message containing the attached file.'}),
                         ('file', ('file:bytes', {}), {
-                            'ro': True,
                             'doc': 'The attached file.'}),
                         ('name', ('file:base', {}), {
                             'doc': 'The name of the attached file.'}),
                     )),
 
                     ('inet:email:message:link', {}, (
-                        ('message', ('inet:email:message', {}), {
-                            'ro': True,
-                            'doc': 'The message containing the embedded link.'}),
                         ('url', ('inet:url', {}), {
-                            'ro': True,
                             'doc': 'The url contained within the email message.'}),
                         ('text', ('str', {}), {
```

Comment:
NB for consistency - docstrings for inet:service:message:link are different (imo inet:email:message:link docstrings are clearer, consider updating inet:service:message:link to be similiar):

('inet:service:message:link', {}, (
      ('title', ('str', {'strip': True}), {
               'doc': 'The title text for the link.'}),
      ('url', ('inet:url', {}), {
               'doc': 'The URL which was attached to the message.'}),
),